src/components/ui/OakLoadingSpinner/OakLoadingSpinner.tsx

```diff
@@ -31,7 +31,6 @@ const StyledLoadingSpinner = styled.span<OakLoadingSpinnerProps>`
   --inner-width: calc(var(--width) / 10 * 8);
   --thickness: calc(var(--width) / 12);
 
-  position: absolute;
```

Comment:
This was throwing off the position of the spinner in a flex container — it doesn't appear to be needed at all looking at all the examples which appear the same.